core/templates/components/common-layout-directives/common-elements/attribution-guide.component.html

```diff
@@ -1,4 +1,4 @@
-<div class="oppia-attribution-guide">
+<div class="oppia-attribution-guide" *ngIf="!checkMobileView()">
   <div [ngStyle]="isLanguageRTL() ? {'left': '25px'} : {'right': '25px'}" class="oppia-cc-icon oppia-transition-200" *ngIf="!deviceUsedIsMobile && !iframed">
```

Comment:
Thanks @krishita30j!! Seems like there's also a `deviceUsedIsMobile` thing here. Is this redundant? Or are they slightly different?

Reply:
Ah, they check different things. `checkMobileView()` also checks if the device is a tablet or not. Whereas `deviceUsedIsMobile` only checks if the device is iPhone or Android.

Reply:
Ah I see -- thanks!